dashboard/app/models/pd/workshop.rb

```diff
@@ -89,6 +106,12 @@ def subject_must_be_valid_for_course
     end
   end
 
+  def all_virtual_workshops_suppress_email
```

Comment:
is this because our communications would be confusing for virtual workshops?

Reply:
Yes, more or less.  You can find this requirement in the table at the very top of [the spec](https://docs.google.com/document/d/1IE_CFnueA4AaAt4CqBAz01jcDRvE7Dgr4jkoZY8krJ0/edit#).  I believe the motivation is that, not only is some of the language in our mailers specific to in-person workshops, but the format and details of virtual workshops will vary more widely and we'd like partners to manage that communication themselves.